common/persistence/src/test/java/app/coronawarn/server/common/persistence/service/DiagnosisKeyServiceTestHelper.java

```diff
@@ -98,7 +98,7 @@ public static DiagnosisKey buildDiagnosisKeyForDateTime(OffsetDateTime dateTime,
    * computed relative to the top of the hours.
    */
   public static DiagnosisKey getKeySubmittedHoursAfterMidnightExpiration(int hours) {
-    LocalDateTime yesterday = LocalDateTime.of(LocalDate.now(), LocalTime.MIDNIGHT).minusDays(1);
+    LocalDateTime yesterday = LocalDateTime.of(LocalDate.now(ZoneId.of("UTC")), LocalTime.MIDNIGHT).minusDays(1);
```

Comment:
consider usage of public statics `ZoneOffset.UTC` instead of fetching UTC with string